docs/index.md

```diff
@@ -1,148 +1,128 @@
-# Integration Guide
+# Integration guide
 
-## API Key
+Pulse displays metrics that provide insights about the current and historic performance of your software delivery process. To calculate these metrics, Pulse must collect information from key events of your particular software development workflow.
 
-The API key identifies your organization and authorizes you to push events to Pulse.
+## Integrating with Pulse
 
-## Pushing Data
+Currently, the recommended way of integrating your workflow with Pulse is by using a CLI to send the necessary data to Pulse.
 
-Currently, we support two ways to push data: a CLI and an HTTP API.
+However, in some scenarios it may not be feasible to use the CLI to send data to Pulse, such as when reporting changes or incidents, or if you are sending data from providers that only support webhooks. For these situations, we provide an HTTP POST webhook on the following URL:
 
-### CLI
+`https://ingestion.acceleratedevops.net/v1/ingestion/<PROVIDER>?api_key=<API KEY>`
```

Comment:
What should go into the `<PROVIDER>` placeholder?

Reply:
@rtfpessoa @DReigada ☝️

Reply:
Nobody should use this without talking with us before.
If you think it is better we can remove it from here.
We will add providers as users need them and then we can document names.
Now we do not support any.

Reply:
So, the `<PROVIDER>` will be provided by us (pardon the pun :sweat_smile:). I will make a quick reference to this.

Reply:
Thinking better about this, I decided to include [this note instead](https://github.com/codacy/pulse-user-docs/pull/5/files#diff-b4d68dc855d0f9476d3f2ee343853bd21bf82ea9960d0cf06661baa244439dd6R13-R14) as it simplifies the current docs, and we'll improve the information on the page as soon as we have the first user asking to use the webhook.